mariadb103.spec

```diff
@@ -1622,6 +1622,9 @@ fi
 %endif
 
 %changelog
+* Thu Jan 09 2020 Andreas Schnederle-Wagner <schnederle@futureweb.at> - 3:10.3.18-2
```

Comment:
The version here in the changelog is incorrect, it should be `3:10.3.21-2`.

Reply:
my bad - overlooked Version, fixed